test/inductor/test_aot_inductor.py

```diff
@@ -110,8 +110,10 @@ def check_model(
     options=None,
     constraints=None,
 ):
-    expected = model(*example_inputs)
-    with config.patch("aot_inductor.abi_compatible", self.abi_compatible):
+    with torch.no_grad(), config.patch(
```

Comment:
I wonder if it's easier to just have a statement in check_model which converts all the inputs to the specific device using .to instead of initializing all of the tensors with self.device in the tests?

Reply:
That's a good idea, unless there is a mixed of input tensor devices, but I guess we can always add another form of `check_model` in that case. Let me update.

Reply:
Hmm, I see there are tests using constraints constructed with a weak ref to tensor, if we convert input tensors in `check_model`, would that cause a problem? I am seeing any problem with local change, but I am not sure it is safe.

Reply:
Ah, I think you might be right, I checked and the IDs of the tensor from .to is different than the original id...